internal/store/elasticsearch/discovery_search_repository.go

```diff
@@ -147,31 +147,34 @@ func (repo *DiscoveryRepository) buildSuggestQuery(ctx context.Context, cfg asse
 	return payload, err
 }
 
-func (repo *DiscoveryRepository) buildTextQuery(ctx context.Context, text string) elastic.Query {
+func (repo *DiscoveryRepository) buildTextQuery(ctx context.Context, cfg asset.SearchConfig) elastic.Query {
 	boostedFields := []string{
```

Comment:
consider this a nit but would suggest to refactor this a bit:

```
	boostedFields := []string{"urn^10", "name^5"}
	if cfg.DisableFuzzy {
		return elastic.NewMultiMatchQuery(cfg.Text, boostedFields...)
	}

	return elastic.NewBoolQuery().Should(
		elastic.NewMultiMatchQuery(cfg.Text, boostedFields...),
		elastic.NewMultiMatchQuery(cfg.Text, boostedFields...).
			Fuzziness("AUTO"),
		elastic.NewMultiMatchQuery(cfg.Text).
			Fuzziness("AUTO"),
	)
```

Reply:
outdated. but have incorporated this - early return in case of flag.